ZooKeepingSystem/Zoo.cs

```diff
@@ -38,19 +50,38 @@ public string ZooName
         /// <param name="numberOfZooKeepers">Total number of zoo keepers employed.</param>
         /// <param name="animals">Animals.</param>
         /// <param name="zooName">Name of the zoo.</param>
+        /// <exception cref="ArgumentNullException">
+        /// Thrown when initialized with no name.
+        /// </exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// Thrown when number of cages or number of zoo keepers is less than zero.
+        /// </exception>
         public Zoo(string zooName, int numberOfCages, int numberOfZooKeepers, Animals animals)
         {
+            if (zooName == null)
+            {
+                throw new ArgumentNullException("Name", "Name cannot be null.");
```

Comment:
- [ ] Change param name from `Name` to `zooName`
- [ ] zooName also should not be blank i.e. `""`